recipes/bimg/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,26 @@
+#include <bx/bx.h>
```

Comment:
Once you have re-used the test_package folder, it will use test_package/test_package.cpp instead of this duplicated file, so you can delete this file.

All this is part of a template that we have been using in CCI and may help you instead of writing for scratch for every new recipe: https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates

Well, your build system is unique, so at least the test package you just need to copy/paste 😅